lib/cell-manager.coffee

```diff
@@ -37,3 +27,39 @@ module.exports = CellManager =
         console.log 'CellManager: Cell [start, end]:', [start, end], 'row:', row
 
         return [start, end]
+
+
+    getBreakpoints: ->
+        editor = atom.workspace.getActiveTextEditor()
+        buffer = editor.getBuffer()
+        breakpoints = [0]
+
+        regex = @getRegexString editor, 'g'
+        if regex?
+            buffer.scan regex, ({range}) ->
+                breakpoints.push range.start.row
+
+        breakpoints.push editor.getLastBufferRow()
+
+        console.log 'CellManager: Breakpoints:', breakpoints
+
+        return breakpoints
+
+
+    getRegexString: (editor, option = '') ->
```

Comment:
What I meant is that this function would be more versatile if it returned a string, and let the caller decide whether they want a Regex and what kind of options the regex should take.